python/lsst/obs/base/ingest_tests.py

```diff
@@ -56,6 +56,11 @@ class IngestTestBase(metaclass=abc.ABCMeta):
     RawIngestTask = lsst.obs.base.RawIngestTask
     """The task to use in the Ingest test."""
 
+    curatedCalibrationDatasetTypes = ()
```

Comment:
There always has to be a writeCuratedCalibrations method because it's required by the abstract base class. I thought the default should be to always call it but not check anything, rather than skipping it. At least then you know it runs.  I can make the default be to skip -- I only added the skipping feature to stop the three separate DECam tests from all testing curated calibrations.